otaserver/app.py

```diff
@@ -28,7 +29,8 @@ def parse_command_line():
            default=__UPLOAD_PATH__,
            help="Path where uploaded files are stored.")
     define("port", default=8080, help="Web application HTTP port.")
-    define("coap_port", default=COAP_PORT, help="CoAP server port.")
+    define("coap_port", default=COAP_PORT, help="CoAP(S) server port.")
+    define("dtls", default=False, help="Enable CoAP Secure.")
```

Comment:
I would rename this option to `use-dtls` or `use-coaps`. What do you think ?

Reply:
Yes, I think that for this application running two services in two different ports, having a more clearly name for all the options related to CoAPS is better.